packs/data/abomination-vaults-bestiary.db/dread-wisp.json

```diff
@@ -286,6 +286,9 @@
                     {
                         "key": "TokenLight",
                         "predicate": {
+                            "all": [
+                                "glowing"
+                            ],
                             "not": [
                                 "go-dark"
                             ]
```

Comment:
Am I missing something or are `glowing` and `go-dark` not the exact same thing? The RollOption is just on another item further down

Reply:
`Dread Flickering` adds the `glowing` and has the `TokenLight` RE on it; `Go Dark` adds `go-dark`, and only has a `RollOption` RE to add the "Go Dark" toggle. I can remove the `toggleable` option and `RollOption` on `Dread Flickering`, which was all I had changed. @TMunsie called out the Dread Wisp as needing the update as well in the Issue, so I was making the assumption that the changes need to be made.